rules/linux/persistence_unusual_sshd_child_process.toml

```diff
@@ -0,0 +1,98 @@
+[metadata]
+creation_date = "2024/12/16"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/12/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the creation of an unusual SSHD child process through the usage of the `new_terms` rule type.
+Attackers may abuse SSH to maintain persistence on a compromised system, or to establish a backdoor for remote access,
+potentially resulting in an unusual SSHD child process being created.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.process*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Unusual SSHD Child Process"
+references = ["https://hadess.io/the-art-of-linux-persistence/"]
+risk_score = 47
+rule_id = "4c3c6c47-e38f-4944-be27-5c80be973bd7"
+severity = "medium"
+tags = [
+    "Domain: Endpoint",
+    "OS: Linux",
+    "Use Case: Threat Detection",
+    "Tactic: Persistence",
+    "Data Source: Elastic Defend"
+]
+timestamp_override = "event.ingested"
+type = "new_terms"
+query = '''
+event.category:process and host.os.type:linux and event.type:start and event.action:exec and
+process.parent.name:(ssh or sshd) and process.args_count:2 and
+not process.command_line:(-bash or -zsh or -sh)
```

Comment:
Can always review telem for FP reduction but some potential ones that I can think of for Cloud, IaaS, etc.

Ansible: `/usr/bin/ansible`
Terraform: `/usr/bin/terraform`
Puppet: `/usr/bin/puppet`
Nessus: `/usr/bin/nessus` (testing related)
Google Cloud SSH Client: `/usr/bin/gcloud` 
Azure SSH Client: `/usr/bin/az`

Reply:
Because this is a new_terms rule I hope these will be filtered out by default based on the 14d new terms condition. I would like to push it without exclusions initially, and tune next quarter there were necessary!